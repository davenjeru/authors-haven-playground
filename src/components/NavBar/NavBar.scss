.nav-logo {
    width: 21%;
    -webkit-border-radius: 10%;
    -moz-border-radius: 10%;
    border-radius: 10%;
}
