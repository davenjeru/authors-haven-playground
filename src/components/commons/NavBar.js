import React from 'react';
import { Link } from 'react-router-dom';

const NavBar = () => (
  <header>
    <nav className="navbar bg-primary fixed-top">
      <div className="container">
        <div className="navbar-brand">
          <Link to="/" className="nav-link text-white">Authors Haven</Link>
        </div>
      </div>
    </nav>
  </header>

);

export default NavBar;
