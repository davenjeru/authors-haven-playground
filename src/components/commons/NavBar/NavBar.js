import React from 'react';
import { Link } from 'react-router-dom';
import AHLogo from '../AHLogo';
import './NavBar.css';

const NavBar = () => (
  <header>
    <nav className="navbar bg-primary fixed-top">
      <div className="container">
        <div className="navbar-brand">
          <Link to="/">
            <AHLogo className="nav-logo" />
          </Link>
        </div>
      </div>
    </nav>
  </header>

);

export default NavBar;
