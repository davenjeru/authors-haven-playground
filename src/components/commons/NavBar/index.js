import React from 'react';
import { Link } from 'react-router-dom';
import AHLogo from '../AHLogo';
import './NavBar.css';
import * as routes from '../../../routes';

const NavBar = () => (
  <header>
    <nav className="navbar bg-primary fixed-top">
      <div className="container">
        <div className="navbar-brand">
          <Link to={routes.INDEX_ROUTE}>
            <AHLogo className="nav-logo" />
          </Link>
        </div>
      </div>
    </nav>
  </header>

);

export default NavBar;
