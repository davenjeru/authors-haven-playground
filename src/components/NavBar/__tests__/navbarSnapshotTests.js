import renderer from 'react-test-renderer';
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import NavBar from '..';

describe('The NavBar', () => {
  it('should not regress', () => {
    const tree = renderer.create(<MemoryRouter><NavBar /></MemoryRouter>);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
