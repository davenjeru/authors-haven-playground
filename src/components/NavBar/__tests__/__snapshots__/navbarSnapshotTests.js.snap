// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The NavBar should not regress 1`] = `
<header>
  <nav
    className="navbar bg-primary fixed-top"
  >
    <div
      className="container"
    >
      <div
        className="navbar-brand"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt="Authors Haven Logo"
            className="nav-logo"
            src="Authors Haven Logo.jpeg"
          />
        </a>
      </div>
    </div>
  </nav>
</header>
`;
